aws/outputs-2.tf

```diff
@@ -4,10 +4,6 @@ output vpc_id {
        value = local.vpc_id
```

Comment:
what is this file outputs-2 for?

Reply:
outputs-2.tf is for outputting the results of magic subnet id and cidr lookups based on names.  It's a separate file because when there was a "vpc" directory for doing development/pre-checks,  it could just be linked in and the  rest of output.tf (which is harder/much-slower to evaluate) could be ignored.

Reply:
I thought about just merging this into outputs.tf now that the vpc dir  is gone.

Reply:
I went ahead and merged outputs-2.tf back into outputs.tf,  there's one file now.